lively.morphic/components/core.js

```diff
@@ -344,7 +344,8 @@ export class ViewModel {
       override = false, converter = false, updater = false
     } of this.bindings) {
       try {
-        if (model) target = this.view.getSubmorphNamed(model).viewModel;
+        // safe navigation allows reuse of view models for UIs with a subset of functionality
+        if (model) target = this.view.getSubmorphNamed(model)?.viewModel;
```

Comment:
I dont know, maybe we can still place a `console.warn` here? Just in case someone does this by accident, or has the name of the input mixed up.

Reply:
I used sloppy language in the original description, sorry. This is actually working as it is now, i.e., I can build "sub-set" components with the "original" `ViewModel`.
What happens is that the line that I changed here fails, which takes us into the `catch` block, leading to a `console.warn`. So we probably have exactly what you wish for? My change only leads to us not entering the `catch` block in the case of a "sub component`".

I see why such a warning can be of help, but found them to become quite spamy. The reason is, that my component is a kind of popup, that gets reinstantiated quite often.

However, I absolutely see why we would keep the warning around, your call. Feel free to close this!

Reply:
I would be open to use the `System.debug` flag in order to limit the warnings we send out for view models. We already do that for quite a bit of warning logs in the system.